mailprocessing/version.py

```diff
@@ -20,4 +20,4 @@
 # Change the version below to change the mailprocessing/imapproc version. This
 # controls the python module's version as well.
 
-PKG_VERSION = "1.2.7"
```

Comment:
Please leave the version at 1.2.7 - that's what I'll be using for the next release (the latest released version is 1.2.6).